### PR TITLE
Adding a Geneva Exporter - Exports to local ETW or UDS

### PR DESCRIPTION
**Pull request by mic-max**
Open Sourcing the OpenTelemetry Geneva Exporter.

Note to reviewers: Please check the code coverage reports, it seems the coverage is lower than expected for the `codecov/project`

### CODE REVIEW

File: examples/owin/Controllers/TestController.cs [hunk 1]
```diff
@@ -14,7 +14,6 @@
 // limitations under the License.
 // </copyright>
 
-using System;
```
Comment (cijothomas) [inline examples/owin/Controllers/TestController.cs:17]: please remove any changes unrelated to this PR's topic.

Comment (mic-max) [reply examples/owin/Controllers/TestController.cs:17]: This was hit by my VS `fix for entire solution`, I will revert all of these.

**Review by cijothomas — APPROVED** [review]

**Review by utpilla — APPROVED** [review]

File: src/OpenTelemetry.Exporter.Geneva/OpenTelemetry.Exporter.Geneva.csproj [hunk 2]
```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <Description>An OpenTelemetry .NET exporter that exports to local ETW or UDS</Description>
+    <Authors>OpenTelemetry Authors</Authors>
+    <NoWarn>$(NoWarn),NU5104,CS1591,SA1123,SA1633,SA1310,CA1031,CA1810,CA1822,CA2000,CA2208,SA1204,SA1201,SA1202,SA1308,SA1309,SA1311,SA1402,SA1602,SA1649</NoWarn>
```
Comment (CodeBlanch) [inline src/OpenTelemetry.Exporter.Geneva/OpenTelemetry.Exporter.Geneva.csproj:7]: 🙈

File: src/OpenTelemetry.Exporter.Geneva/OpenTelemetry.Exporter.Geneva.csproj [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    <TargetFrameworks Condition="$(OS) == 'Windows_NT'">$(TargetFrameworks);net461</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="OpenTelemetry" Version="1.2.0-rc2" />
+  </ItemGroup>
+
+  <PropertyGroup>
+    <EnableNETAnalyzers>true</EnableNETAnalyzers>
+    <AnalysisLevel>latest</AnalysisLevel>
```
Comment (CodeBlanch) [inline src/OpenTelemetry.Exporter.Geneva/OpenTelemetry.Exporter.Geneva.csproj:18]: Just FYI in this repo you should be able to just do...

https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/99967016d9cb3a6d76a95cae1c270f2bed5c6687/src/OpenTelemetry.Extensions/OpenTelemetry.Extensions.csproj#L9

Defined here:

https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/99967016d9cb3a6d76a95cae1c270f2bed5c6687/build/Common.targets#L3

File: src/OpenTelemetry.Exporter.Geneva/OpenTelemetry.Exporter.Geneva.csproj (hunk 2 above)
Comment (mic-max) [reply src/OpenTelemetry.Exporter.Geneva/OpenTelemetry.Exporter.Geneva.csproj:7]: Ya, the code style in the other repository was a bit different than this one. I figured ignoring most of the harder to fix warnings was the best way to go for now, and eventually work on converging the rules with the repo as a whole :)

**Pull request merged**